### ISSUE TITLE
[request] Publish docker image on alternative registries

### ISSUE THREAD

**Issue by tomasged** [alpine-docker/multi-arch-docker-images#4]
Given the pull limits of Docker Hub, it's a bit unfortunate to pull the image from docker.com.

Either the project gets the "Sponsored OSS" status ( https://www.docker.com/community/open-source/application/ ) or the images are better published within registries with increased API limits like the public ecr from AWS.

Right now, (my case) when using testcontainers people will find the rate limit issue: https://java.testcontainers.org/supported_docker_environment/image_registry_rate_limiting/

Thanks!

**Comment by ozbillwang** [comment]
Do you recommend any other registries?

The main issue I’m facing is that I can no longer find an simple project name(such as `alpine`) available in any of the popular registries.

**Comment by Murazaki** [comment]
Can you just publish here ? (on ghcr.io)

You should be able to package to different names from one repo.

Here is an example and the Github Actions that does it : 
https://github.com/orgs/games-on-whales/packages?repo_name=gow
https://github.com/games-on-whales/gow/blob/master/.github/workflows/docker-build-and-publish.yml

**Comment by ozbillwang** [comment]
yes, but the new images would be 'alpine-docker/xxxx', not looks good. that's why i don't choice

**Comment by Murazaki** [comment]
It would be better than not having alternatives though. you can still look for another name or repo after that.

BTW you can also push to :
- quay.io
- gallery.ecr.aws
- gitlab.com

**Comment by ozbillwang** [comment]
Thanks for your advice, but I have no time to maintain another registry.

**Comment by tomasged** [comment]
Hi @ozbillwang , now that the possibility of publishing to another registry is discarded, would it be possible for ask for the "Sponsored OSS" status? That solves the issue for everybody. Thanks.

**Comment by ozbillwang** [comment]
they are all "Sponsored OSS"

![Image](https://github.com/user-attachments/assets/356c2933-d256-4321-b3a1-2f999b8af036)

**Comment by tomasged** [comment]
Oh, great! Then the situation got improved. Thank you!
